georocket-server/src/main/java/io/georocket/GeoRocket.java

```diff
@@ -613,18 +615,22 @@ protected static JsonObject configureGeoRocket() {
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    JsonObject conf = configureGeoRocket();
+    try {
```

Comment:
You only need to put the `configureGeoRocket` call in the try-catch block. The rest can be outside the block.